udata_front/theme/gouvfr/datagouv-components/src/api/tabularApi.ts

```diff
@@ -0,0 +1,44 @@
+import { config } from "../config";
+
+export type SortConfig = {
+  column: string;
+  type: string;
+} | null;
+
+/**
+ * Fetch url
+ */
+export function fetchData(url: string) {
+  return fetch(url)
+    .then(res => {
+      if (!res.ok) {
+        throw new Error('Network error');
+      }
+      return res.json();
+    })
+    .then(data => {
+      return { data };
+    });
+}
```

Comment:
This function seems really generic, could it be useful to have it in an `helpers` file instead of `tabularApi.ts`?

Reply:
Now that we are in udata-front, we can even use axios like the rest of the codebase